approximate_baseline/approximateBandpasses.py

```diff
@@ -0,0 +1,35 @@
+import os
+import numpy as np
+from lsst.utils import getPackageDir
```

Comment:
I *think* DM wants to reserve the right to change the name of the environment variables set up by EUPS whenever they want (and then adjust `getPackageDir` accordingly).

It just occurred to me that my `__file__` scheme won't work, since this script is meant to be run from within `approximate_baseline/`.  Could we use `os.environ['PWD']`?

Also: there should be some code to prevent the user from accidentally overwriting the baseline throughputs, just in case they try to run this script from outside of `approximate_baseline/`.  Just check that `throughputfile != fname`.  If they are equal, through an exception.